scenarios/Assistants/assistant-bot-in-a-box/README.md

```diff
@@ -39,7 +39,7 @@ The flow of messages is as follows:
 
 ```
 git clone https://github.com/Azure/AI-in-a-Box
```

Comment:
repo path is out of date?

Reply:
the repo is fine, but they changed the folder of this repo